### PR TITLE
Mandy C - Lesson 1.3. Create and use a new component to show info cards

### PR DESCRIPTION
**Pull request by m4ndycheung**

### CODE REVIEW

File: src/TouristInfoCards.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+function TouristInfoCards() {
+  const touristCardData = [
+    {
+      image:
+        "https://images.pexels.com/photos/12319481/pexels-photo-12319481.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1",
+      city: "Glasgow",
+      description:
+        "Ancestor laugh fat lit chair running Uruks master loyal ready. Explanation Dori aside. Frailty within Witch-king greedily. What about second breakfast? ",
```
Comment (shieldo) [inline src/TouristInfoCards.js:10]: Ahhhh Lord Of The Rings Lorem Ipsum!! I hadn't heard of this before :)

File: src/TouristInfoCards.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    },
+  ];
+
+  return touristCardData.map((element) => (
+    <div className="card">
+      <div className="card-body">
+        <img src={element.image} className="card-img-top" />
+        <h2 className="card-title">{element.city}</h2>
+        <p className="card-description">{element.description}</p>
+        <a href="#" className="btn btn-primary">
```
Comment (shieldo) [inline src/TouristInfoCards.js]: It looks like this needs to use the `url` property too.

**Review by shieldo — CHANGES_REQUESTED** [review]

File: src/TouristInfoCards.js (hunk 1 above)
Comment (m4ndycheung) [reply src/TouristInfoCards.js:10]: It's awsome!!

File: src/TouristInfoCards.js (hunk 2 above)
Comment (m4ndycheung) [reply src/TouristInfoCards.js]: THanks for spotting this, Douglas, I've now added this property

**Pull request merged**